src/IncrementalInference.jl

```diff
@@ -506,6 +507,14 @@ export
   DataLayerAPI
 
 
+#TreeClique
+const TreeClique = Graphs.ExVertex
+
+DFG.getLabel(cliq::Graphs.ExVertex) = cliq.attributes["label"]
+function setLabel(cliq::Graphs.ExVertex, lbl::String)
```

Comment:
This one still uses `cliq::Graphs.ExVertex`. Should it be changed as well for `cliq::TreeClique`?